src/openssh/patch/0001-Put-style-as-line-number-to-ssh-session-environment-.patch

```diff
@@ -0,0 +1,28 @@
+From 839f98d579e3a22b6ca075fd026eee55bdb2ef25 Mon Sep 17 00:00:00 2001
+From: Blueve <672454911@qq.com>
+Date: Mon, 26 Oct 2020 06:44:59 +0000
+Subject: [PATCH] Put style as line number to ssh session environment variable
+
```

Comment:
can you put following description in this patch. this is from your design doc.

```
By default, the content between : and @ will be trimmed by sshd before it do authentication and the trimmed string will be dropped silently. To use this segment as line number for reverse SSH feature, we need to modify the source ode of OpenSSH and put this segment to a environment variable SSH_TARGET_CONSOLE_LINE, then we can insert a short script into /etc/bash.bashrc and run command consutil connect $SSH_TARGET_CONSOLE_LINE to enter the management session automatically after user login.
```

Reply:
Sure, I will update them